video/out/hwdec/hwdec_vaapi.c

```diff
@@ -177,6 +179,7 @@ static int init(struct ra_hwdec *hw)
     p->ctx->hwctx.hw_imgfmt = IMGFMT_VAAPI;
     p->ctx->hwctx.supported_formats = p->formats;
     p->ctx->hwctx.driver_name = hw->driver->name;
+    p->ctx->hwctx.conversion_filter_name = conversion_filter_name;
```

Comment:
Not that it really matters but any particular reason to not just do `p->ctx->hwctx.conversion_filter_name = "scale_vaapi";`?

Reply:
Allergy to inline strings, but you're right that it's fine here.